adr/cli.py

```diff
@@ -19,13 +19,21 @@
 log.addHandler(logging.StreamHandler())
 
 
+def print_to_file(data, filepath):
+    f = open(filepath, "x")
```

Comment:
Let's just use 'w' here, otherwise it'll be annoying to run locally.

Reply:
Thanks I updated :)